src/test/java/net/minecraftforge/debug/client/TestScreen.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.client;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+public class TestScreen extends Screen
+{
+    public static int open() {
+        Minecraft.getInstance().setScreen(new TestScreen(Component.translatable("screen.patch")));
```

Comment:
Formatting: braces on newlines. 

Additionally, use a literal component here with readable text, since there is no definition for the `screen.patch` translation key.

```suggestion
    public static int open()
    {
        Minecraft.getInstance().setScreen(new TestScreen(Component.literal("Test screen")));
```